vta/include/vta/hw_spec.h

```diff
@@ -32,6 +31,12 @@ extern "C" {
 
 #include <stdint.h>
 
+/*! Memory bus width */
+#define VTA_BUS_WIDTH (1 << VTA_LOG_BUS_WIDTH)
+
+/*! Register file width */
+#define VTA_REG_WIDTH 32
```

Comment:
Is `VTA_REG_WIDTH` derived from `LOG_ACC_WIDTH` or has some relationship to that?

Reply:
I think we can remove it for now. The idea was to do the accumulation in narrower bits (e.g. 24) since for most operations it would be sufficient. However, loading biases at 24 bits is difficult, so the memory transfer would happen in 32bits, but it would be stored and operated upon internally in 24bits. I'll clean it up for now since it can bring confusion. We can add it later if need be.